src/release/release.yml

```diff
@@ -0,0 +1,29 @@
+# Orb Version 0.0.1
+
+version: 2.1
+description: Wrapper for Artsy's release management tools
+
+executors:
+  scripts:
+    docker:
+      - image: artsy/orb-scripts
+
+jobs:
+  block:
+    executor: scripts
+    parameters:
+      project_id:
+        type: integer
+        description: The primary key of the project object in horizon
+    steps:
+      - run:
+          description: Check horizon to see if there's an unresolved deployment block
+          command: |
+            RESULT=$(curl -s -f -u ${HORIZON_USER}:${HORIZON_PASS} "https://releases.artsy.net/api/deploy_blocks?project_id=<< parameters.project_id >>&resolved=false")
+            if [ "$RESULT" == "[]" ]; then
```

Comment:
Is there any chance this could fail on an incorrect project_id?

Reply:
Good point @eessex ... other failure cases would be good to lie out there, including horizon being down or other responses

Reply:
If `project_id` is invalid the service returns a 404. It would fail the build, but it wouldn't trigger this `if`. 

Definitely something to test further/improve on though.

Reply:
Not a blocker, but agreed it seems like other error cases are worth exploring.

Reply:
I believe the `[-f / --fail` option](https://curl.haxx.se/docs/manpage.html#-f) is intended to provide support for when horizon is down is "Fail silently (no output at all) on server errors.".

Suffice to say, I think some light-weight tests with something like [bats](https://github.com/sstephenson/bats) would be helpful to document how this script works and to confirm our assumptions.